### DISCUSSION TITLE
dashboard 配置的证书请求有问题，不知道哪里问题，求指导

### DISCUSSION THREAD

**Discussion by scoolor** [apache/apisix#8532] [Q&A]
<img width="1095" alt="image" src="https://user-images.githubusercontent.com/20853694/208241035-e58202bb-75eb-4964-ad4b-0a409bb92231.png">

**Comment by juzhiyuan** [comment]
Hi @scoolor, I have no idea but can you first Google `failed to find any SSL certificate by SNI`? There have a lot of related cases may help you.

**Comment by tokers** [comment]
@scoolor It's not sufficient to answer your question since whether a TLS handshake can be done also decided by the TLS SNI. So you need to tell us how you send requests to APISIX, also, the certificate parsed details is helpful.
